mobile/lib/core/configuration.dart

```diff
@@ -661,6 +662,14 @@ class Configuration {
     await _preferences.setBool(hasSelectedAllFoldersForBackupKey, value);
   }
 
+  Future<void> setShouldLoopVideo(bool value) async {
```

Comment:
Let's move this to localSettings? The key value can be `video.loop` or at least with `video.should_loop`